includes/discovery/mempools/quanta.inc.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/*
+ * LibreNMS Quanta memory information module
+ *
+ * Copyright (c) 2017 Mark Guzman <segfault@hasno.info>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+if ($device['os'] == "quanta") {
+    d_echo('Quanta Memory:');
+    //QUANTA-LB6M-REF-MIB::agentSwitchCpuProcessMemFree
+    $avail = snmp_get($device, '.1.3.6.1.4.1.4413.1.1.1.1.4.1.0', '-Oqv');
```

Comment:
You should use the name and MIB seeing as you have it. Same goes for throughout the code, makes reading what it's doing easier.